### PR TITLE
GeoServer 2.21.1 release anouncement

### PR DESCRIPTION
**Pull request by jodygarnett**
available for review

### CODE REVIEW

File: _posts/2022-08-01-geoserver-2-21-1-released.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+tags:
+- Release
+release: release_221
+version: 2.21.1
+jira_version: 16856 
+---
+
+GeoServer [2.21.1](/release/2.21.1/) release is now available with downloads ([bin](https://sourceforge.net/projects/geoserver/files/GeoServer/2.21.1/geoserver-2.21.1-bin.zip/download), [war](https://sourceforge.net/projects/geoserver/files/GeoServer/2.21.1/geoserver-2.21.1-war.zip/download), [windows](https://sourceforge.net/projects/geoserver/files/GeoServer/2.21.1/GeoServer-2.21.1-winsetup.exe/download)), along with [docs](https://sourceforge.net/projects/geoserver/files/GeoServer/2.21.1/geoserver-2.21.1-htmldoc.zip/download) and [extensions](https://sourceforge.net/projects/geoserver/files/GeoServer/2.21.1/extensions/).
+
+This is a stable releaes of the GeoServer 1.21.x series, made in conjunction with GeoTools 27.1
```
Comment (mprins) [inline _posts/2022-08-01-geoserver-2-21-1-released.md:17]:
```suggestion
This is a stable release of the GeoServer 1.21.x series, made in conjunction with GeoTools 27.1 
```

File: _posts/2022-08-01-geoserver-2-21-1-released.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+* Number of items held in the resource cache is shown, so there is visual feed back when using **Clear** button.
+* Documentation has been updated to cover all the status field descriptions and document the available actions
+
+For more information see [Server Status page](https://docs.geoserver.org/stable/en/user/configuration/status.html#server-status).
+
+![Server status](/img/posts/2.21/server_status.png) <br/>
+
+### JVM Console
+
+A new **JVN Console** tab has been added to the server status page allowing a summary of memory use to be reviewed and downloaded, and a summary of active threads to be reviewed and downloaded.
```
Comment (mprins) [inline _posts/2022-08-01-geoserver-2-21-1-released.md:36]:
```suggestion
A new **JVM Console** tab has been added to the server status page allowing a summary of memory use to be reviewed and downloaded, and a summary of active threads to be reviewed and downloaded.
```

**Review by mprins — CHANGES_REQUESTED** [review]
I found some typos

**Comment by jodygarnett** [comment]
> I found some typos

I am in transit - please fix typos and merge.

**Pull request merged**